nlp_architect/utils/utils_squad.py

```diff
@@ -0,0 +1,1017 @@
+# copied from huggingface library, adapted imports
```

Comment:
some of the methods in this file should be moved to `data` and some are relevant only to the squad model and should be attached to the model code under `models` package.

Reply:
moved compute_softmax() and get_best_indexes() to data/utils.py